### ISSUE TITLE
spacy build fails on pip v18.1

### ISSUE THREAD

**Issue by cyrilchemparathy** [opensemanticsearch/open-semantic-search#333]
Running a docker build bombs out on spacy, with the following error message:
        WARNING: pip versions <19.3 (currently installed: 18.1) are unable to detect binary wheel compatibility for blis. To avoid a source install with a very long compilation time, please upgrade pip with `pip install --upgrade pip`.

The indicated resolution works.

```
diff --git a/Dockerfile b/Dockerfile
index 27c4dff..571b202 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -13,6 +13,8 @@ RUN adduser --system --disabled-password spacy
 
 COPY displacy /usr/lib/python3/dist-packages/spacy-services/displacy
 
+RUN pip3 install --upgrade pip
+
 # install dependencies like SpaCy and language specific models
 RUN pip3 install --no-cache-dir -r /usr/lib/python3/dist-packages/spacy-services/displacy/requirements.txt
```

**Comment by boan-anbo** [comment]
thanks. fixed mine

> Running a docker build bombs out on spacy, with the following error message:
> WARNING: pip versions <19.3 (currently installed: 18.1) are unable to detect binary wheel compatibility for blis. To avoid a source install with a very long compilation time, please upgrade pip with `pip install --upgrade pip`.
> 
> The indicated resolution works.
> 
> ```
> diff --git a/Dockerfile b/Dockerfile
> index 27c4dff..571b202 100644
> --- a/Dockerfile
> +++ b/Dockerfile
> @@ -13,6 +13,8 @@ RUN adduser --system --disabled-password spacy
>  
>  COPY displacy /usr/lib/python3/dist-packages/spacy-services/displacy
>  
> +RUN pip3 install --upgrade pip
> +
>  # install dependencies like SpaCy and language specific models
>  RUN pip3 install --no-cache-dir -r /usr/lib/python3/dist-packages/spacy-services/displacy/requirements.txt
> ```

thanks. fixed mine

**Comment by Mandalka** [comment]
Thank you very much for bugreport and the solution PR.

I merged it, since new Debian 10.7 yet has the old pip version:
https://packages.debian.org/buster/python3-pip

When Debian bullseye (currently "testing"-release) will become the new "stable"-release we can hopefully get rid of this additional stuff:
https://packages.debian.org/bullseye/python3-pip

**Comment by Mandalka** [comment]
Added same workaround to DEB package, so no manual step there, too, especially important for automatic builds of VMs like Open Semantic Desktop Search:

https://github.com/opensemanticsearch/spacy-services.deb/commit/0ef5548e0c31e258f4b4e0c034c537489a07a8fd
